### DISCUSSION TITLE
failing to add a comment

### DISCUSSION THREAD

**Discussion by Dieterbe** [pwntester/octo.nvim#217] [Q&A]
Hi, i use neovim 0.5.1 and trying to use the latest version of this plugin.

I'm failing to add a comment to an issue
with `:Octo issue list` i can load issue, and open one.

Now, the readme says:

> Usage
> Just edit the issue title, body or comments as a regular buffer and use :w(rite) to sync the issue with GitHub.

So, I thought i would be able to append to the bottom of the issue document.
But when i hit 'o' on the last line to go into insert mode, everything just hangs, not responsive to any keypress. I have to force quit neovim.

I also tried the alternative way, with the same issue loaded, I run `:Octo comment add`, add some text and save.  But on github, the comment never appears.

Looks like a neat plugin, but seems broken for me.
thanks,
Dieter

**Comment by pwntester** [comment]
> So, I thought i would be able to append to the bottom of the issue document.
But when i hit 'o' on the last line to go into insert mode, everything just hangs, not responsive to any keypress. I have to force quit neovim.

I cant reproduce the hanging. Note that you can modify any existing comments in the issue/pr (any text that is marked within the lines in the signcolumn. You will see how the line go red when you modify it and grow when you add new lines)

To add new comments, you need to call `:Octo comment add`. Adding new lines at the end of the buffer will not add new comments.

> But on github, the comment never appears.

Did you get any errors? Do you have `gh` installed and authenticated (`gh auth login`)?

**Reply by Dieterbe** [reply]
strange, now `:Octo comment add` seems to work. not sure why. i had my gh installed and authenticated, and now it just works.  perhaps something went wrong with the api call the first time? (but then wouldn't the plugin show an error of some kind?)
